buildenv/jenkins/variables/defaults.yml

```diff
@@ -286,163 +200,71 @@ s390x_linux_xl:
-    13: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-openssl=fetched'
-    next: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-openssl=fetched'
+    all: '--with-cups-include=/opt/freeware/include'
+    8: ' --disable-ccache'
+    11: '--disable-warnings-as-errors'
+    13: '--disable-warnings-as-errors'
+    14: '--disable-warnings-as-errors'
+    next: '--disable-warnings-as-errors'
   build_env:
     vars:
       8: 'PATH+XLC=/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin'
       11: 'PATH+XLC=/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin'
-      12: 'PATH+XLC=/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin'
       13: 'PATH+XLC=/opt/IBM/xlC/16.1.0/bin:/opt/IBM/xlc/16.1.0/bin CC=xlclang CXX=xlclang++'
+      14: 'PATH+XLC=/opt/IBM/xlC/16.1.0/bin:/opt/IBM/xlc/16.1.0/bin CC=xlclang CXX=xlclang++'
       next: 'PATH+XLC=/opt/IBM/xlC/16.1.0/bin:/opt/IBM/xlc/16.1.0/bin CC=xlclang CXX=xlclang++'
   excluded_tests:
     11:
       - special.system
-    12:
-      - special.system
-    13:
-      - special.system
 #========================================#
 # Linux x86 64bits Compressed Pointers
 #========================================#
 x86-64_linux:
+  extends: ['openssl']
   boot_jdk:
     8: '/usr/lib/jvm/jdk-7'
     11: '/usr/lib/jvm/jdk-11'
-    12: '/usr/lib/jvm/jdk-11'
     13: '/usr/lib/jvm/adoptojdk-java-12'
+    14: '/usr/lib/jvm/adoptojdk-java-13'
     next: '/usr/lib/jvm/adoptojdk-java-13'
   release:
+    all: 'linux-x86_64-server-release'
     8: 'linux-x86_64-normal-server-release'
     11: 'linux-x86_64-normal-server-release'
-    12: 'linux-x86_64-server-release'
-    13: 'linux-x86_64-server-release'
-    next: 'linux-x86_64-server-release'
   freemarker: '/home/jenkins/freemarker.jar'
   openjdk_reference_repo: '/home/jenkins/openjdk_cache'
   node_labels:
-    build:
-      8: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-      11: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-      12: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-      13: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-      next: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-  extra_getsource_options:
-    8: '--openssl-version=1.1.1d'
-    11: '--openssl-version=1.1.1d'
-    12: '--openssl-version=1.1.1d'
-    13: '--openssl-version=1.1.1d'
-    next: '--openssl-version=1.1.1d'
+    build: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
   extra_configure_options:
-    8: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
-    11: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
-    12: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched'
-    13: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched'
-    next: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched'
+    all: '--enable-cuda --with-cuda=/usr/local/cuda-9.0'
```

Comment:
This should include `--enable-jitserver`, assuming #8191 is merged first.

Reply:
Based on https://github.com/eclipse/openj9/pull/8191#issuecomment-571298582 I will leave as-is for now. Please advise if you'd like it included or if 8191 is merged first.